### DISCUSSION TITLE
Tags Autocompletition

### DISCUSSION THREAD

**Discussion by cupcakearmy** [directus/directus#3486] [Archive]
I was wondering if there is a way to autocomplete already used tags.
Also is there a way of getting all the possible used in a collection, or do i simply query all the objects and make a set?

**Comment by rijkvanzanten** [comment]
No and no. The current `tags` interface is a "dumb" (not-relational) interface that saves the value as JSON or CSVs into the DB. Directus isn't aware of which ones are used. 

There's an open PR to implement a type-ahead style M2M interface which sounds like what you're looking for 🙂  https://github.com/directus/directus/pull/2899

**Reply by cupcakearmy** [reply]
Perfect, gonna wait until this is done then 👍 
Any ETA?

**Reply by rijkvanzanten** [reply]
I'm pretty busy wrapping up some last things before the holidays, but if you'd be able to review that PR and see if it works as expected on your end that would help speed this up tremendously 🙂

**Reply by benhaynes** [reply]
Posting this here for future reference:

https://docs.directus.io/getting-started/support.html#can-you-give-an-eta-for-this-feature-fix
